demos/wasi/postman-echo/k8s.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: postman-echo
+  labels:
+    app: postman-echo
+  annotations:
+    alpha.wasi.krustlet.dev/allowed-domains: '["https://postman-echo.com"]'
```

Comment:
Is this a normal YAML array?

Reply:
What do you mean "normal"?

Reply:
Looking at the example, I am not sure how to format the YAML value for multiple allowed domains.

Looking at the YAML spec for sequences / collections (https://yaml.org/spec/1.2/spec.html#id2759963), I thought this would be similar to:

```
key:
  - string1
  - string2      
  - string3
  - string4
```
or

```
key: ['string1', 'string2', 'string3', 'string4']
```

This is basically me not understanding how to write YAML.

Reply:
Oh yes. I agree. Annotations in k8s is a map of key-value pairs, presumably of type string:string. So we can't unmarshal a traditional `yaml` list. I.e. The Kubernetes annotation type we are (un)marshaling is "map[string]string" and a `list` to `string` conversion in yaml is not supported AFAIK....Then again, it is yaml.

Reply:
So we have to serialize the list as a string, that makes sense.

Ok, thanks for the clarification!